codespell_lib/_codespell.py

```diff
@@ -301,6 +301,19 @@ def _split_lines(self, text: str, width: int) -> List[str]:
         return out
 
 
+def _toml_to_parseconfig(toml_dict: Dict[str, Any]) -> Dict[str, Any]:
+    """Convert a dict read from a TOML file to the parseconfig.read_dict() format."""
+    parseconfig_dict = {}
+    for key, value in toml_dict.items():
+        if value is False:
+            pass
+        elif value is True:
+            parseconfig_dict[key] = ""
+        else:
+            parseconfig_dict[key] = value
+    return parseconfig_dict
```

Comment:
Also I'm not sure there's a reason not to have this operate in-place
```suggestion
def _toml_to_parseconfig(toml_dict: Dict[str, Any]) -> None:
    """Convert a dict read from a TOML file to the parseconfig.read_dict() format."""
    for key, value in toml_dict.items():
        if value is False:
            pass
        elif value is True:
            toml_dict[key] = ""
        else:
            toml_dict[key] = value
```

Reply:
It's just that you cannot modify `data["codespell"]` directly in place. You have to:
* `d = data["codespell"]`
* modify `d` in place
* `data["codespell"] = d`